charts/prometheus-statsd-exporter/values.yaml

```diff
@@ -76,6 +76,15 @@ serviceMonitor:
   # More info https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
   relabelings: []
 
+podMonitor:
```

Comment:
Thanks @monotek for the suggestion.
I think it is a good idea to combile the config for both serverMonitor and podMonitor.

However, I will prefer to set [2 flags to enable serviceMonitor and podMonitor saparately](https://github.com/prometheus-community/helm-charts/pull/2596/files#diff-5e6d51ef06ed55367a5c12de02ad1caccc4bd17568b4d1ec97d90d5aa11d1c04R69). Because it it needed for the migration from self-hosted prometheus to Google Managed Prometheus, At some points we still need to maintainance 2 monitor systems at the same time until all the services fully migrated to GMP (at least this is the scenario we are facing now)